distributed/tests/test_utils.py

```diff
@@ -554,6 +555,22 @@ async def test_offload():
     assert (await offload(lambda x, y: x + y, 1, y=2)) == 3
 
 
+@pytest.mark.asyncio
+async def test_offload_preserves_contextvars():
```

Comment:
Thanks for putting this together @gjoseph92. For reference, this test passes for me on `main`

Reply:
Good catch, thanks for checking.